packages/js-moi-utils/dist/index.d.ts

```diff
@@ -1,17 +1,16 @@
+export { Hex } from "../types/hex";
+export { AssetCreationReceipt, AssetMintOrBurnReceipt, LogicDeployReceipt, LogicInvokeReceipt } from "../types/receipt";
+export * from "./address";
+export * from "./asset";
+export * from "./base64";
 export * from "./bytes";
 export * from "./errors";
-export * from "./base64";
 export * from "./hex";
-export * from "./asset";
 export * from "./interaction";
+export * from "./json";
+export * from "./object";
 export * from "./properties";
-export * from "./address";
 export * from "./schema";
-export * from "./object";
-export * from "./hex";
-export * from "./json";
-export { Hex } from "../types/hex";
-export { AssetCreationReceipt, AssetMintOrBurnReceipt, LogicDeployReceipt, LogicInvokeReceipt } from "../types/receipt";
-export { Tesseract } from "../types/tesseract";
-export { Interaction } from "../types/interaction";
+export { Interaction, Participants, Participant } from "../types/interaction";
 export { Receipt } from "../types/receipt";
+export { Tesseract } from "../types/tesseract";
```

Comment:
nitpick, order the export statements neatly

Reply:
68dbb0137444a2ca86b44860f6459f4dba7c4ef6